LayoutTests/compositing/scrolling/fixed-style-with-rotation.html

```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
```

Comment:
You need to have these tests run as "text" tests. Add:
```
if (window.testRunner)
  testRunner.dumpAsText();
```

and you also need to include the resulting `.txt` files in your PR.

Reply:
Done.

Reply:
For future reference, unless a test case requires invalid HTML to reproduce a bug, it's usually best to structure documents like this:

```
<!DOCTYPE html>
<html>
<head>
<script>[as needed]</script>
<style>[as needed]</style>
</head>
<body>
[content goes here, but can also include script tags]
</body>
</html>
```

Reply:
Thank you ! I will keep this in mind.